testdata/pods/two_container_sleep.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
Where is this file used ?

Reply:
https://github.com/openshift/cucushift/pull/7577

Reply:
Then move the test file to cucushift, along with the scenario.

Reply:
updated, please help to merge